tests/test_lattice.py

```diff
@@ -1,5 +1,6 @@
 from unittest.mock import patch
 import logging
+from typing import List, Set, Tuple
```

Comment:
we dont need to add type annotation for test functions, though it is okay to have them